.golangci.yml

```diff
@@ -0,0 +1,2 @@
+service:
```

Comment:
Why this file was introduced? We have .golangci.yml in backend directory, this was not supposed to be introduced.

Reply:
I had the same concern and almost deleted the file as a dup, but I believe this was addes not for our linter but for the external one run by GolangCI

Reply:
I suggest adding a comment about this file purpose then.

Reply:
if this file is needed for real I'd rather move the one from the `backend` to root.

Reply:
@smaant would you be so kind as to clarify the purpose of this file?

Reply:
Resolving in favor of #615.